<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Car Form</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>989</width>
         <height>637</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="8" column="0">
         <widget class="QLabel" name="lablePrice">
          <property name="font">
           <font>
            <family>PMingLiU-ExtB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Price:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lableCountry">
          <property name="font">
           <font>
            <family>PMingLiU-ExtB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Country:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="inputYear">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>2004</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2003</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2002</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2001</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1999</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1998</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1997</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1996</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1995</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1994</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1993</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLineEdit" name="inputModel">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QComboBox" name="inputCountry">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>USA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Canada</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mexico</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QComboBox" name="inputMake">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Chevy</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dodge</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ford</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jeep</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Honda</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nissan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Acura</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Toyota</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lableYear">
          <property name="font">
           <font>
            <family>PMingLiU-ExtB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Year:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lableModel">
          <property name="font">
           <font>
            <family>PMingLiU-ExtB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Model:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="10">
         <widget class="QTextEdit" name="displayData">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Georgia</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>ForbiddenCursor</cursorShape>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="markdown">
           <string/>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Georgia'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lableMake">
          <property name="font">
           <font>
            <family>PMingLiU-ExtB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Make:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLineEdit" name="inputPrice">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="submitBtn">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
